package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "nearley",
-  "version": "2.16.0",
+  "name": "@mongodb-forks/nearley",
+  "version": "1.0.0",
```

Comment:
How would you feel about `0.1.0` as our version starting out? I don't want other people to expect API compatibility from us.